### PR TITLE
Fix local test failures in flutter_tools

### PR DESCRIPTION
**Pull request by Hixie**
- Don't wrap text in tests (otherwise we'll be sensitive to the terminal width).
- Minor clean-up in terminal.dart

### CODE REVIEW

**Review by dnfield — APPROVED** [review]

**Review by christopherfujino — APPROVED** [review]
LGTM, thanks for this

File: packages/flutter_tools/test/src/context.dart [hunk 1]
```diff
@@ -82,7 +82,7 @@ void testUsingContext(
             when(mock.getAttachedDevices()).thenAnswer((Invocation _) async => <IOSSimulator>[]);
             return mock;
           },
-          OutputPreferences: () => OutputPreferences(showColor: false),
+          OutputPreferences: () => OutputPreferences(showColor: false, wrapText: false),
```
Comment (zanderso) [inline packages/flutter_tools/test/src/context.dart]: Unfortunately, we have two ways of creating a context for testing. The other one is in testbed.dart, and should probably get a similar update.

Comment (blasten) [reply packages/flutter_tools/test/src/context.dart]: luckily it's in the OKRs :)

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/41863

**Review by zanderso — APPROVED** [review]

**Comment by Hixie** [comment]
landing with flutter-build red because as far as i can tell the tree is actually green and flutter-build is wrong.

**Pull request merged**